docs/source/index.mdx

```diff
@@ -140,32 +142,139 @@ _another_ subgraph. For example, the `Review` type includes a
+
+<ExpansionPanel title="Click to expand">
+
+```graphql
+schema
+  @core(feature: "https://specs.apollo.dev/core/v0.1"),
+  @core(feature: "https://specs.apollo.dev/join/v0.1")
+{
+  query: Query
+}
+
+directive @core(feature: String!) repeatable on SCHEMA
+
+directive @join__field(graph: join__Graph, requires: join__FieldSet, provides: join__FieldSet) on FIELD_DEFINITION
+
+directive @join__type(graph: join__Graph!, key: join__FieldSet) repeatable on OBJECT | INTERFACE
+
+directive @join__owner(graph: join__Graph!) on OBJECT | INTERFACE
+
+directive @join__graph(name: String!, url: String!) on ENUM_VALUE
+
+scalar join__FieldSet
+
+enum join__Graph {
+  ACCOUNTS @join__graph(name: "accounts" url: "http://localhost:4001")
+  PRODUCTS @join__graph(name: "products" url: "http://localhost:4002")
+  REVIEWS @join__graph(name: "reviews" url: "http://localhost:4003")
+}
+
+type Product
+  @join__owner(graph: PRODUCTS)
+  @join__type(graph: PRODUCTS, key: "upc")
+  @join__type(graph: REVIEWS, key: "upc")
+{
+  name: String! @join__field(graph: PRODUCTS)
+  price: Int @join__field(graph: PRODUCTS)
+  reviews: [Review] @join__field(graph: REVIEWS)
+  upc: String! @join__field(graph: PRODUCTS)
+}
+
+type Query {
+  me: User @join__field(graph: ACCOUNTS)
+  topProducts(first: Int = 5): [Product] @join__field(graph: PRODUCTS)
+}
+
+type Review {
+  author: User @join__field(graph: REVIEWS, provides: "username")
+  body: String
+  product: Product
+}
+
+type User
+  @join__owner(graph: ACCOUNTS)
+  @join__type(graph: ACCOUNTS, key: "id")
+  @join__type(graph: REVIEWS, key: "id")
+{
+  id: ID! @join__field(graph: ACCOUNTS)
+  reviews: [Review] @join__field(graph: REVIEWS)
+  username: String! @join__field(graph: ACCOUNTS)
+}
+```
+
+</ExpansionPanel>
+
+As you can see, the supergraph schema includes a lot of federation-specific additions! These additions are used only by the gateway, and you'll never need to add them manually.
+
+### API schema
+
+The gateway uses its [supergraph schema](#supergraph-schema) to produce an _API schema_, which is what's exposed to clients as your actual GraphQL API. This schema cleanly and logically represents the combination of your [subgraph schemas](#subgraph-schemas):
+
+<ExpansionPanel title="Click to expand">
+
+```graphql
+type Product {
+  name: String!
+  price: Int
+  reviews: [Review]
+  upc: String!
+}
+
+type Query {
+  me: User
+  topProducts(first: Int = 5): [Product]
+}
+
+type Review {
+  author: User
+  body: String
+  product: Product
+}
+
+type User {
+  id: ID!
+  reviews: [Review]
+  username: String!
+}
+```
+
+</ExpansionPanel>
 
 ## Gateway example
 
-The gateway for our federated data graph fetches the schema from each
-subgraph and composes those schemas into a single graph:
+The gateway for a federated data graph provides its composed [supergraph schema](#supergraph-schema) to the `ApolloGateway` constructor:
+
+```js{4}
+const supergraphSchema = readFileSync('./supergraph.graphql').toString();
```

Comment:
Nit: rename this variable to `supergraphSdl` and shorthand it below. In code, `sdl` and `schema` do have somewhat distinct meanings, and this is definitely sdl.